src/engine/x86-64/X86_64Interpreter.v3

```diff
@@ -2552,6 +2570,14 @@ class X86_64InterpreterGen(ic: X86_64InterpreterCode, w: DataWriter) {
 			genExtractLane(Opcode.F32X4_EXTRACTLANE, BpTypeCode.F32.code, byte.!(4), asm.movd_r_m, asm.movd_m_r, false);
 			genExtractLane(Opcode.F64X2_EXTRACTLANE, BpTypeCode.F64.code, byte.!(8), asm.movq_r_m, asm.movq_m_r, false);
 		}
+		{	// V128 lane replacement
+			genReplaceLane(Opcode.I8X16_REPLACELANE, byte.!(1), asm.movb_r_m, asm.movb_m_r);
```

Comment:
The `byte.!` casts shouldn't be necessary, as the compiler will allow literals within range to be bytes.